runtime/bcutil/ROMClassBuilder.cpp

```diff
@@ -355,6 +356,48 @@ ROMClassBuilder::handleAnonClassName(J9CfrClassFile *classfile, bool *isLambda)
 	return result;
 }
 
+BuildResult
+ROMClassBuilder::patchConstantPool(J9CfrClassFile *classfile, ROMClassCreationContext *context) {
+	J9CfrConstantPoolInfo* constantPool = classfile->constantPool;
+	U_16 i = 0;
+	BuildResult result = OK;
+	PORT_ACCESS_FROM_PORT(_portLibrary);
+	J9VMThread *currentThread = context->currentVMThread();
+
+	j9array_t array = (j9array_t)J9_JNI_UNWRAP_REFERENCE(context->getCPPatch());;
+	j9object_t item = NULL;
+	jsize arrayBound = J9INDEXABLEOBJECT_SIZE(currentThread, array);
+
+	for (U_16 i = 0; i < arrayBound; i++) {
+		item = J9JAVAARRAYOFOBJECT_LOAD(currentThread, array, i);
+		if (item != NULL) {
+			switch(constantPool[i].tag) {
+			case CFR_CONSTANT_Utf8:
+				/* NYI - requires extra buffer array for utf8 longer than original */
+				/* Do nothing for UTF8 at the moment
+				constantPool[i].slot1 = 0;
+				constantPool[i].bytes = NULL;
+				*/
+				break;
+			case CFR_CONSTANT_Integer:
```

Comment:
Are these cases actually used?  If not, I would rather not support them if we can avoid it